assets/openai.yaml

```diff
@@ -418,11 +418,11 @@
   # General
   organization: OpenAI
   description: ''
-  created_date: 2019-11
+  created_date: Nov 2019
```

Comment:
I forget, but do partial dates break the sorting. If we don't know the day of month, I think it is fine to say Nov 1, 2019 and include in descritpion we assumed to be Nov 1

Reply:
I don't think they break sorting, they are automatically placed at the end of that month (for MM-YY) or year for (YY).

Yes that would work, will update all the partial dates accordingly